app/build.gradle.kts

```diff
@@ -77,7 +77,7 @@ android {
 }
 
 dependencies {
-    implementation(project(":core:design-system"))
+    implementation(projects.core.designSystem)
```

Comment:
Is this on purpose?

I see this on `master`.
![image](https://github.com/user-attachments/assets/89c7d799-b421-491f-b315-ced5650ef3b4)